docs/source/nbconvert_library.ipynb

```diff
@@ -79,7 +79,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "The nbformat API returns a special type of dictionary. For this example, you don't need to worry about the details of the structure (if you are interested, please see the [nbformat documentation](https://nbformat.readthedocs.io/en/latest/)).\n",
+    "The nbformat API returns a special type of dictionary. For this example, you don't need to worry about the details of the structure (if you are interested, please see the [nbformat documentation](https://nbformat.readthedocs.io/en/latest/) ).\n",
```

Comment:
This change shouldn't have to be made. It's only here because the link check gets the link wrong.

Reply:
This is now fixed by using nbsphinx properly in 3832b3c